### PR TITLE
Ordering upcoming workshops by date, soonest first

### PR DESCRIPTION
**Pull request by gvwilson**
Once this is merged, the home page dashboard will show upcoming events with the soonest at the top.

### CODE REVIEW

File: workshops/models.py [hunk 1]
```diff
@@ -204,9 +204,12 @@ def upcoming_events(self):
         '''Return a queryset for all published upcoming events.
 
         Upcoming events are those which start after today and are published.
+        Events are ordered by date, soonest first.
         '''
 
-        queryset = self.filter(start__gt=datetime.date.today()).filter(published=True)
+        queryset = self.filter(start__gt=datetime.date.today())\
+                        .filter(published=True)\
+                        .order_by('start')
```
Comment (pbanaszkiewicz) [inline workshops/models.py]: According to my PEP8 linter, lines 211-212 should be unindented 1 space.

Comment (gvwilson) [inline workshops/models.py]: Sorry, editor glitch - fix pushed.

**Comment by gvwilson** [comment]
See also #338.

**Comment by pbanaszkiewicz** [comment]
Looks OK.

**Pull request merged**